src/app/zones/components/MultizoneInfo.vue

```diff
@@ -45,14 +46,9 @@ export default {
 
   data() {
     return {
+      env,
```

Comment:
**Tip**: In Options API components, you can set non-reactive properties on the top level of the default export (e.g. beneath `name: 'MultizoneInfo',`) and subsequently access them in the template using `$options.env`.

This is better because setting `env` in the `data` function will turn it into a reactive object which we don’t want.

Reply:
Oh nice! Will take look at that now, ty!

Reply:
I moved both `env` and `productName` (that was also hanging on `data`) to the root of the exported object, not super keen on the magical global-like `$options` thing, but I suppose we'll eventually convert this to composition API at some point, so it will go.

Reply:
Yeah, that’s one of the reasons I strongly prefer composition API style components because almost all things are explicit (i.e. no magic `this.$*` things).